csrc/scheduler/normalization_inner_outer.cpp

```diff
@@ -846,18 +1152,35 @@ std::unique_ptr<ReductionParams> getInnerOuterPersistentHeuristics(
       hp.threads_per_block_min,
       hp.threads_per_block_max);
 
-  std::unique_ptr<ReductionParams> rparams = innerOuterPersistentHeuristic(
-      properties.total_iteration_numel,
-      properties.total_reduction_numel,
-      buffer_params.regs_buffer_size,
-      buffer_params.smem_buffer_size,
-      buffer_params.smem_overhead,
-      max_outer_reduction_dtype_size,
-      hp.vectorize_factor,
-      hp.threads_per_block_min,
-      hp.threads_per_block_max,
-      buffer_params.project_to_input,
-      runtime_info.getIndexType());
+  std::unique_ptr<ReductionParams> rparams;
+
+  if (isOptionEnabled(EnableOption::WarpSpecializedPersistent)) {
```

Comment:
IIUC, our plan is to have two scheduler co-exist and choose the better one with a heuristics.

Nitpick, can we put a comment here on the absence of heuristics to choose the right scheduler? Otherwise the code reads like we are in transition from one to the other.

Reply:
Goot point! Added.
```
  // Ultimately, we want the heuristic to decide between using the
  // warp-specialized version or the multi-wave version. The enable option is a
  // temporary configuration to facilitate testing during development without
  // disrupting existing behavior.
````